azkaban-common/src/main/java/azkaban/project/ProjectManager.java

```diff
@@ -196,21 +181,13 @@ public Props getProps() {
   }
 
     /**
-     * Checks if a project is active using project_name
-     *
-     * @param name
-     */
-    public Boolean isActiveProject(String name) {
-        return projectsByName.containsKey(name);
-    }
-
-    /**
      * Checks if a project is active using project_id
      *
      * @param name
      */
     public Boolean isActiveProject(int id) {
```

Comment:
Looks like this method is not used any longer? Who will call it ?Should we just remove it? Besides, the method name sounds a bit confused here. We should detect exist here, rather than active, right?